demo/play-demo-for-recording.sh

```diff
@@ -0,0 +1,208 @@
+   -i fruit-client-sb \
+   -g me.fruitstand \
+   -p me.fruitstand.demo \
+   -s 2.1.6.RELEASE \
+   -t client \
+   -v 1.0.0-SNAPSHOT \
+   --supported=false  \
+   fruit-client-sb
+sleep 3
+
+clear && sleep 1
+echo "# Create a backend maven project named fruit-backend-sb interactively making sure you use crud as the template type"|pv -qL 10
+sleep 3
+exec hal component spring-boot fruit-backend-sb
+sleep 3
+
+clear && sleep 1
+echo "# Build the projects"|pv -qL 10
+echo "# Compile and generate the uber jar of the Spring Boot client application"|pv -qL 10
+sleep 3
+exec mvn package -f fruit-client-sb
+sleep 3
+
+clear && sleep 1
+echo "# Repeat the command executed previously for the CRUD - backend microservice."|pv -qL 10
+echo "# We need to use the kubernetes profile because the project is set up to work both locally using H2 database for quick testing and \"remotely\" using a PostgreSQL database."|pv -qL 10
+sleep 4
+exec mvn package -f fruit-backend-sb -Pkubernetes
+sleep 3
+
+clear && sleep 1
+echo "# Deploy the applications as components"|pv -qL 10
+sleep 2
+exec hal component create -c fruit-client-sb
+sleep 2
+exec hal component create -c fruit-backend-sb
+sleep 3
+
+clear && sleep 1
+echo "# Check that the components have been correctly installed"|pv -qL 10
+sleep 2
+exec oc get cp
+sleep 3
+
+clear && sleep 1
+echo "# Create a capability to install a PostgreSQL database using the interactive mode"|pv -qL 10
+sleep 2
+exec hal capability create -n postgres-db
+sleep 3
+
+clear && sleep 1
+echo "# Check the capability status"|pv -qL 10
+sleep 2
+exec oc get capabilities
+sleep 10
+
+clear && sleep 1
+echo "# Link the microservices"|pv -qL 10
+echo "# We need to wire the fruit-backend-sb component with the postgres-db capability by creating a link between them"|pv -qL 10
+sleep 4
+exec hal link create -n backend-to-db -t fruit-backend-sb
+sleep 3
+
+clear && sleep 1
+echo "# Now, create a link targeting the fruit-client-sb component to wire the client and backend"|pv -qL 10
+sleep 3
+exec hal link create -n client-to-backend -t fruit-client-sb -e KUBERNETES_ENDPOINT_FRUIT=http://fruit-backend-sb:8080/api/fruits
+sleep 3
+
+clear && sleep 1
+echo "# Check the link status"|pv -qL 10
+sleep 2
+exec oc get links
+sleep 3
+
+clear && sleep 1
+echo "# Push the code"|pv -qL 10
+sleep 2
+exec hal component push -c fruit-client-sb,fruit-backend-sb
+echo "# Let's wait a few seconds to let maven build the application within the pod and start the application"|pv -qL 10
+#Wait some seconds for pods readies
+i=0
+while [ $i -le 15 ]
+do
+    printf '. '
+    i=$(( $i + 1 ))
+    sleep 1
+done
+
+
+clear && sleep 1
+echo "# Call the REST endpoint of the Fruit backend service to verify if we can access it"|pv -qL 10
+echo "# Obtain the route address of the backend microservice using this command "|pv -qL 10
+sleep 4
+exec oc get routes/fruit-backend-sb --template={{.spec.host}}
+echo " "
+sleep 2
+echo "# Let's create some fruits ;-)"|pv -qL 10
```

Comment:
As you use `httpie` here, then you should mention it --> `Let's create some fruits on the backend using the HTTPie - https://httpie.org tool`
and remove it later -> line 196